lib/broadside/deploy.rb

```diff
@@ -4,9 +4,11 @@ class Deploy
 
     attr_reader :target
 
+    delegate :tag, to: :target
+
     def initialize(options = {})
       @target = Broadside.config.get_target_by_name!(options[:target])
-      @tag = options[:tag] || @target.tag
+      @target.tag = options[:tag] if options[:tag]
```

Comment:
> i'm of the opinion maybe we should just reference @tag here:
https://github.com/lumoslabs/broadside/blob/master/lib/broadside/ecs/ecs_deploy.rb#L31
instead of @target.tag.

i think i agree with @matl33t - _definitely_ that block should reference `@tag`, and if that simplfies other things as well then awesome.